remoulade/state/backends/postgres.py

```diff
@@ -0,0 +1,114 @@
+import datetime
+from typing import Any, List
+
+from pytz import utc
+from sqlalchemy import Column, DateTime, SmallInteger, String, asc, create_engine, desc, inspect
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from remoulade.state import State, StateBackend
+
+Base: Any = declarative_base()
```

Comment:
I added the Any type to avoid having a mypy error when declaring StoredState as child class of 'Base'. After reading about this issue on the mypy github I didn't find any better solution.